backend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "backend",
+    "version": "1.0.0",
+    "description": "",
+    "main": "src/app.ts",
+    "scripts": {
+        "tsc": "tsc",
+        "dev": "ts-node-dev --no-deps --respawn --poll --interval 1000 src/app.ts",
+        "prod": "node -r ./build/app.js",
+        "tslint": "tslint -p tsconfig.json",
+        "prettier": "prettier --config .prettierrc --write ./src/**/*.{ts,tsx,js,jsx}"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "dependencies": {
+        "body-parser": "^1.19.0",
+        "dotenv": "^8.0.0",
+        "express": "^4.17.1",
+        "mysql": "^2.17.1",
```

Comment:
Why not postgress?

Reply:
I needed simple database, might upgrade later